internal/cmd/prune.go

```diff
@@ -0,0 +1,34 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2019 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func newdockerPruneCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Pruning",
```

Comment:
`Use` if the command, so this would lead to `iofogctl Pruning ...` Please change to `prune`

Reply:
Updated to the cmd use.